serving/docker/lmi.Dockerfile

```diff
@@ -32,7 +32,7 @@ ARG datasets_version=2.20.0
 ARG autoawq_version=0.2.5
 ARG tokenizers_version=0.19.1
 # LMI-Dist Deps
-ARG vllm_wheel="https://github.com/vllm-project/vllm/releases/download/v0.5.3.post1/vllm-0.5.3.post1-cp310-cp310-manylinux1_x86_64.whl"
+ARG vllm_wheel="s3://djl-ai-staging/publish/vllm/cu124-pt231/vllm-0.5.3.post1+cu124-cp310-cp310-linux_x86_64.whl"
```

Comment:
@lanking520 do we have cloudfront url for gamma bucket?

Reply:
https://publish.djl.ai/vllm/cu124-pt231/vllm-0.5.3.post1+cu124-cp310-cp310-linux_x86_64.whl